### PR TITLE
ESM entry in psammead-assets

### PR DESCRIPTION
**Pull request by dr3**
Contributes to #557

**Overall change:** Adds ESM support to psammead-assets

**Code changes:**

- Uses ESM modules package key
- Restructures package to use nested `package.json`s to expose ESM code
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]

File: packages/utilities/psammead-assets/index.test.jsx [hunk 1]
```diff
@@ -21,29 +17,32 @@ const svgsExpectedExports = {
...
   });
 
-  it('should test all the utility exports exist and are the correct type when coming from src/', () => {
+  it('should test all the utility exports exist and are the correct type for module', () => {
     testUtilityPackages(
-      actualExportsFromSrc,
+      getExports('module'),
       expectedExports,
-      'psammead-assets',
+      'psammead-assetss',
```
Comment (thekp) [inline packages/utilities/psammead-assets/index.test.jsx]: spelling mistake ?

Comment (dr3) [reply packages/utilities/psammead-assets/index.test.jsx]: Fixed :)

**Review by thekp — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]
Noice

**Review by thekp — APPROVED** [review]

**Comment by dr3** [comment]
Doesnt need testing, will regression any issues

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**